scripts/brc4/update_database_release.pl

```diff
@@ -55,9 +56,15 @@
 
 ###############################################################################
 sub get_databases {
-  my ($dbh) = @_;
+  my ($dbh, $prefix) = @_;
   
-  my $db_array = $dbh->selectcol_arrayref("SHOW DATABASES;");
+  my $db_array;
+  if ($prefix) {
+    say("Search dbs with prefix $prefix:");
```

Comment:
```suggestion
    say("Search dbs with prefix '$prefix':");
```
Just a tiny detail (I find having the text between `'` helps when mistakenly passing blank spaces).